helm-chart/kuberay-apiserver/Chart.yaml

```diff
@@ -15,10 +15,4 @@ type: application
 # This is the chart version. This version number should be incremented each time you make changes
 # to the chart and its templates, including the app version.
 # Versions are expected to follow Semantic Versioning (https://semver.org/)
-version: 0.3.0
-
-# This is the version number of the application being deployed. This version number should be
-# incremented each time you make changes to the application. Versions are not expected to
```

Comment:
Do we want to use helm version to track the upgrade in future?

Reply:
Could you elaborate?

The `version` field is still there.